From 59ee87b6065255375d8fb1f5ec4fb93202e93018 Mon Sep 17 00:00:00 2001
From: David Heidelberg <david.heidelberg@collabora.com>
Date: Sat, 21 Oct 2023 15:58:20 +0200
Subject: [PATCH 656/834] ci/freedreno: disable Adreno 660 Vulkan pre-merge

No need to move it to manual/nightly - we already run full runs there.

Revert when Adreno 660 gets stable again!

Reported-by: Helen Koike <helen.koike@collabora.com>
Signed-off-by: David Heidelberg <david.heidelberg@collabora.com>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/25839>
---
 src/freedreno/ci/gitlab-ci.yml | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/freedreno/ci/gitlab-ci.yml b/src/freedreno/ci/gitlab-ci.yml
index b150c658a89..32d9da37b38 100644
--- a/src/freedreno/ci/gitlab-ci.yml
+++ b/src/freedreno/ci/gitlab-ci.yml
@@ -155,7 +155,8 @@ a660_gl:
   variables:
     DEQP_SUITE: freedreno-a660
 
-a660_vk:
+# disabled due to 1.3.7.0 CTS uprev, we have already -full job running in nightly
+.a660_vk:
   extends:
     - .lava-test-deqp:arm64
     - .collabora-turnip-rules
@@ -167,7 +168,7 @@ a660_vk:
 
 a660_vk_full:
   extends:
-    - a660_vk
+    - .a660_vk
     - .collabora-turnip-manual-rules
   parallel: 3
   timeout: 3h
-- 
2.42.0

